### ISSUE TITLE
add nats support! ▼▼▼

### ISSUE THREAD

**Issue by stormkiller0** [WiIIiam278/HuskSync#288]
add nats support! ▼▼▼

 advantages of Nats:
  - high speed
  - convenient cluster creation

 nats repository: https://github.com/nats-io/nats-server

**Comment by ProdPreva1l** [comment]
I don't believe we have plans to add any other forms of messaging services, if you fancy high speed I can recommend [KeyDB by Snapchat](https://github.com/Snapchat/KeyDB)

**Comment by WiIIiam278** [comment]
I'm not actually opposed to adding rabbit or this or whatever, wouldn't be a major refactor - provided the db has effectively the same feature set of redis and the overheads are fairly minimal on our side
